bootstrap-app/src/cmr/bootstrap/services/dispatch/core_async_dispatch.clj

```diff
@@ -0,0 +1,143 @@
+(ns cmr.bootstrap.services.dispatch.core-async-dispatch
```

Comment:
Which means I get to:
1. keep protocols organized together for large projects (makes a great reference, and it's nice to know where they all like)
1. still use protocols in Clojure like they are meant to (the engine of APIs that have multiple implementations), in particular:
1. I get to `(require '[proj.some.ns.thing.core :as thing-api])` and then call `(thing-api/func ...)` because I used potemkin to import the vars defined in the protocol itself. And since  this is where the `extend`s happen, all of the implementations are treated as one from the API user's perspective (since extends dispatches based on type, under the hood; never a need to go poking into `impl`, iow).